src/audio.cpp

```diff
@@ -222,6 +222,20 @@ namespace openre::audio
         snd_se_on(a0, nullptr);
     }
 
+    // 0x004EC450
+    void snd_load_core(uint8_t id, uint8_t arg1)
```

Comment:
Here you can use: 
`interop::call<void, uint8_t, uint8_t>(0x004EC450, id, arg1);`